docs/development/writingzeppelininterpreter.md

```diff
@@ -1,173 +0,0 @@
----
-layout: page
-title: "Writing Zeppelin Interpreter"
```

Comment:
Are you sure that this needs to be removed as part of R interpreter implementation?